lego/apps/joblistings/views.py

```diff
@@ -45,8 +47,12 @@ def get_serializer_class(self):
         return JoblistingSerializer
 
     def get_queryset(self):
-        if self.action == "list":
-            return Joblisting.objects.filter(
+        queryset = Joblisting.objects.all()
+
+        time_filter = self.request.query_params.get("timeFilter", False)
```

Comment:
Yes now that I think of it, it makes more sense to set the default to True hehe

Reply:
My opinion has changed. I think the default should be `False`. Firstly, if the default is `True`, we have to do a dumb `time_filter == True` comparison because the query param is a string. But, most importantly it prevents older joblistings from being viewed, as the queryset will always be filtered even when fetching a single joblisting that falls outside the filter.